src/ethereum_test_tools/common/types.py

```diff
@@ -556,6 +562,7 @@ class Environment:
     withdrawals: Optional[List[Withdrawal]] = None
     parent_data_gas_used: Optional[int] = None
     parent_excess_data_gas: Optional[int] = None
+    parent_beacon_block_root: Optional[bytes] = None
```

Comment:
On the newest main, this definition would change to:
```
    parent_beacon_block_root: Optional[FixedSizeBytesConvertible] = field(
        default=None,
        json_encoder=JSONEncoder.Field(
            name="parentBeaconRoot",
            cast_type=Hash,
        ),
    )
```

Reply:
We should rename this field to `beacon_block_root` in order to not confuse this field with others that come from the parent execution block.